service/internal/logger/audit/logger.go

```diff
@@ -15,10 +15,32 @@ const (
 	LevelAuditStr = "AUDIT"
 )
 
+var logLevelNames = map[slog.Leveler]string{
+	LevelAudit: LevelAuditStr,
+}
+
 type Logger struct {
 	logger *slog.Logger
 }
 
+// Used to support custom log levels showing up with custom labels as well
+// see https://betterstack.com/community/guides/logging/logging-in-go/#creating-custom-log-levels
+func CustomReplaceAttr(_ []string, a slog.Attr) slog.Attr {
```

Comment:
When is this function used? Should it be given a name that reflects how or why it will be used? This name feels like a placeholder

Reply:
It's used as the `ReplaceAttr` function passed into the slog logger (see link in comment). I can update the name to say that it's like the audit level creator or something along those lines